txpipe/utils/calibration_tools.py

```diff
@@ -67,6 +67,19 @@ def apply_metacal_response(R, S, g1, g2):
     return mcal_g[:,0], mcal_g[:,1]
 
 
+def apply_lensfit_calibration(g1, g2, weight=None,m=None):
+    #This function needs to be checked!
+    
+    if weight==None:
+        weight = np.array([1])
+    if m==None:
+        m = np.array([1])
+
+    return g1, g2
```

Comment:
Did you mean to delete all the stuff that was here?

Reply:
It definitely looks wrong now :-)

Reply:
Hi Joe. Yes, it was wrong. Emily told me she committed this version by accident and I thought she was correcting me. Anyway, the old version that I suggested it was not right either. But now, I checked some emails exchange we had with Catherine and I'm adding a new version that it seems a bit more right. It is based on this suggestion that Catherine gave to us: _"On m-corrections per galaxy - these do exist but they are pretty much nonsense....For TXPipe, I would suggest you simply attach a responsivity of 1 per galaxy, and then the actual m-correction per tomographic bin folds through as a non-zero mean m nuisance parameter prior per bin with the width of 0.02 (kannawadi et al 2019 has the values to use for the peak of the prior). We have 100% correlation between the m-errors in each bin - can TXPipe/FireCrown deal with correlated nuisance parameters?"_

So, my interpretation of this was: don't apply the m-correction (a.k.a the 1+K term) and just pass to treeCorr the lensfit weight. Although, I let the possibility to pass the individual m.